sample-hacep-project/sample-hacep-project-client/src/main/java/org/kie/hacep/sample/client/ClientProducerDemo.java

```
@@ -50,14 +50,13 @@ private static void insertBatchEvent(int items) throws IOException {
 
     private static Properties getProperties() {
         Properties props = CommonConfig.getStatic();
-        // for openshift 3.11 minishift -> props.put("bootstrap.servers", "my-cluster-kafka-bootstrap-my-kafka-project.<ip>.nip.io:443");
-        // for openshift 4.X  crc -> props.put("bootstrap.servers", "my-cluster-kafka-bootstrap-my-kafka-project.apps-crc.testing:443");
-        props.put("bootstrap.servers", "<kafka_bootstrap_exposed>:443");
-        props.put("security.protocol", "SSL");
-        props.put("ssl.keystore.location", "/<path>/openshift-drools-hacep/sample-hacep-project/sample-hacep-project-client/src/main/resources/keystore.jks");
-        props.put("ssl.keystore.password", "<password>");
-        props.put("ssl.truststore.location", "/<path>/openshift-drools-hacep/sample-hacep-project/sample-hacep-project-client/src/main/resources/keystore.jks");
-        props.put("ssl.truststore.password", "<password>");
+
+        try (InputStream is = ClientProducerDemo.class.getClassLoader().getResourceAsStream("configuration.properties")) {
+            props.load(is);
+        } catch (IOException io) {
+            io.printStackTrace();
```

Can you please propagate the exception instead of just print it?